sBTC SIP

```diff
@@ -0,0 +1,166 @@
+# sBTC SIP
+
+## SIP Number: 022
+**Title:** A Decentralized Two-Way Bitcoin Peg  
+**Type:** Standard  
+**Status:** Draft  
+
+## Abstract
+sBTC is a novel digital asset that lets you move Bitcoin in and out of the Stacks blockchain. With sBTC, users can interact with Clarity smart contracts, which enable Bitcoin applications such as payments, decentralized lending, decentralized exchanges, and BTC-backed stablecoins.
+
+sBTC is a SIP-010 token on the Stacks blockchain, backed 1:1 against BTC, and operated by a decentralized set of signers. When BTC is locked on the Bitcoin L1, an equivalent amount of sBTC is issued on the Stacks layer, ensuring a consistent 1:1 ratio of sBTC:BTC. Users can redeem their sBTC at any time by submitting a withdrawal request. Once the request is processed by sBTC signers, BTC is returned to the user’s specified address on the Bitcoin L1.
```

Comment:
Recommend dropping the layerist wording, since it doesn't add anything.

"When BTC is locked on the Bitcoin L1" --> "When BTC is locked on Bitcoin."

"an equivalent amount of sBTC is issued on the Stacks layer" --> "an equivalent amount of sBTC is issued on Stacks"

Also, you might want to mention that the sBTC is destroyed when it is redeemed for real BTC.